### PR TITLE
Get questions

### PR DESCRIPTION
**Pull request by mathildegrimal**
#### Before submitting, make sure:

- [x] You have rebased your branch
- [x] You describe your changes
- [x] You describe How to test
- [ ] I work on the same file as one of my colleague which one?

#### Describe below your changes:
Added route and methods to get all the questions and corresponding answers from the API

##### Describe below How to test:
send a postman GET request on http://localhost:3001/ekh2o/questions/getAllQuestions and see the magic !

### CODE REVIEW

**Comment by Antonin-dev** [comment]
i like this magic
postman => ok

File: src/Questions/questions.router.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * @url /ekh2o/questions/getAllQuestions
+ * @method get
+ * @argument   
+ */
+router.get(
+  '/getAllQuestions',
+
+  async (req, res, next) => {
+    try {
+      logger('info', `${Path.questions}getQuestions begin`);
```
Comment (Antonin-dev) [inline src/Questions/questions.router.ts:20]: /getAllQuestions

File: src/Questions/questions.router.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      
+      const questions = await getAllQuestions();
+      logger(
+        'success',
+        `${Path.questions}getQuestions`,
```
Comment (Antonin-dev) [inline src/Questions/questions.router.ts:25]: /getAllQuestions

**Review by Antonin-dev — APPROVED** [review]

**Review by mathildegrimal — COMMENTED** [review]
Updates done after your comments @Antonin-dev

**Pull request merged**